src/activities/quizzes/QuizEntity.js

```diff
@@ -217,6 +256,10 @@ export class QuizEntity extends Entity {
 		return disableRightClick !== this.isDisableRightClickEnabled();
 	}
 
+	_hasDisablePagerAndAlerts(disablePagerAndAlerts) {
```

Comment:
should probably be `_hasDisablePagerAndAlertsChanged` since this checks if it's changed